docs/core/tutorials/using-mstest-on-windows.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Use MSTest with .NET Core on Windows
+description: How to use MSTest with .NET Core on Windows, using Visual Studio 2015
+keywords: MSTest, .NET, .NET Core
+author: 
+manager: 
+ms.date: 08/18/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 
```

Comment:
Please generate a guid here at guidgenerator.com, and add it here. (We need this for the build process.)